client/js/options.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
 const $ = require("jquery");
+const _ = require("lodash");
```

Comment:
How do you suggest to fix that? Duplicate the functionality? Or can I require("lodash.escaperegexp") ?